lib/resque-loner/unique_job.rb

```diff
@@ -24,13 +24,18 @@ def redis_key(payload)
           payload = decode(encode(payload)) # This is the cycle the data goes when being enqueued/dequeued
           job  = payload[:class] || payload['class']
           args = (payload[:args]  || payload['args'])
-          args.map! do |arg|
-            arg.is_a?(Hash) ? arg.sort : arg
-          end
+          args = map_args(args)
 
           digest = Digest::MD5.hexdigest(encode(class: job, args: args))
```

Comment:
I'd probably rename the helper method `encode_args`, or maybe `canonicalize_args`, because that describes what it does instead of how it does it.

Then this would become:

```suggestion
          digest = Digest::MD5.hexdigest(encode({ class: job, args: encode_args(args) }))
```